superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -146,12 +146,14 @@ class SliceHeaderControls extends React.PureComponent {
             disabled={this.props.chartStatus === 'loading'}
           >
             {t('Force refresh')}
-            <div
-              className="refresh-tooltip"
-              data-test="dashboard-slice-refresh-tooltip"
-            >
-              {refreshTooltip}
-            </div>
+            {refreshTooltip && (
+              <div
+                className="refresh-tooltip"
+                data-test="dashboard-slice-refresh-tooltip"
```

Comment:
seems like this could be really easy to add a unit test for since this already has a `data-test` attribute. Might be worth adding a test for when the dashboard is refreshing then the div isn't rendered

Reply:
will work on a test later